PowerEditor/installer/nsisInclude/globalDef.nsh

```diff
@@ -29,10 +29,43 @@
 ; Define the application name
 !define APPNAME "Notepad++"
 
-!define APPVERSION "7.5.1"
-!define APPNAMEANDVERSION "${APPNAME} v${APPVERSION}"
-!define VERSION_MAJOR 7
-!define VERSION_MINOR 51
+; ------------------------------------------------------------------------
+; Get Notepad++ version from the notepad++ binary
+
+!ifdef ARCH64
+	!getdllversion "..\bin64\notepad++.exe" nppVer_
+!else
+	!getdllversion "..\bin\notepad++.exe" nppVer_
+!endif
+
+!define APPVERSION		${nppVer_1}.${nppVer_2}		; 7.5
```

Comment:
I just don't see how nppVer_1, nppVer_2, nppVer_3 & nppVer_4 are asigned.

Reply:
They are assigned by `!getdllversion`.

Reply:
Right, it is assigned by ```!getdllversion```

From [nsis doc](http://nsis.sourceforge.net/Reference/!getdllversion)
>!getdllversion
>
>This is similar to GetDLLVersionLocal, only it stores the version number in defines and can therefore be used anywhere, not just inside functions and sections.
>
>!getdllversion "$%windir%\explorer.exe" expv_
>!echo "Explorer.exe version is ${expv_1}.${expv_2}.${expv_3}.${expv_4}"